designer/processReports/src/main/scala/pl/touk/nussknacker/processCounts/influxdb/InfluxConfig.scala

```diff
@@ -3,7 +3,10 @@ package pl.touk.nussknacker.processCounts.influxdb
 import pl.touk.nussknacker.engine.util.metrics.common.naming.{nodeIdTag, scenarioIdTag}
 import sttp.client3.UriContext
 
-case class InfluxConfig(influxUrl: String, user: Option[String], password: Option[String],
+case class InfluxConfig(influxUrl: String,
+                        user: Option[String],
+                        password: Option[String],
+                        additionalHeaders: Map[String, String] = Map.empty,
```

Comment:
That would be more complicated, Sttp's API looks like this:
![obraz](https://github.com/TouK/nussknacker/assets/1409807/3d9fe653-f84c-4780-ba9f-547945cedc17)
Right now I can just pass entire `additionalHeaders` into it.

Reply:
converting `List[(String, String)]` into `Seq[Header]` doesn't look like sth too complicated ;)

Reply:
Then it would need to look like this in configuration:
```
additionalHeaders: [
  {name: "X-Grafana-Org-Id", value: "12"},
  {name: "Multi-Valued-Header", value: "a"},
  {name: "Multi-Valued-Header", value: "b"},
]
```
Unless you know a better way of modeling it? Starts to look complicated ;)

I can do it that way if you want, but IMO it just looks complex for no reason - user can easily join `Multi-Valued-Header` into single entry when writing configuration.

Reply:
OK, I understand. So, I agree it's not worth to do it.